checks.d/mongo.py

```diff
@@ -61,6 +58,84 @@ class MongoDb(AgentCheck):
         "metrics.repl.buffer.count",
         "metrics.repl.buffer.maxSizeBytes",
         "metrics.repl.buffer.sizeBytes",
+
+        # mongo 3.x metrics
+        "locks.Collection.acquireCount.R",
+        "locks.Collection.acquireCount.W",
+        "locks.Collection.acquireWaitCount.R",
+        "locks.Collection.acquireWaitCount.W",
+        "locks.Collection.timeAcquiringMicros.R",
+        "locks.Collection.timeAcquiringMicros.W",
+        "locks.Database.acquireCount.r",
+        "locks.Database.acquireCount.w",
+        "locks.Database.acquireWaitCount.r",
+        "locks.Database.acquireWaitCount.w",
+        "locks.Database.timeAcquiringMicros.r",
+        "locks.Database.timeAcquiringMicros.w",
+        "locks.Global.acquireCount.r",
+        "locks.Global.acquireCount.w",
+        "locks.Global.acquireWaitCount.r",
+        "locks.Global.acquireWaitCount.w",
+        "locks.Global.timeAcquiringMicros.r",
+        "locks.Global.timeAcquiringMicros.w",
+        "locks.Metadata.acquireCount.R",
+        "locks.Metadata.acquireCount.W",
+        "locks.MMAPV1Journal.acquireCount.r",
+        "locks.MMAPV1Journal.acquireCount.w",
+        "locks.MMAPV1Journal.acquireWaitCount.r",
+        "locks.MMAPV1Journal.acquireWaitCount.w",
+        "locks.MMAPV1Journal.timeAcquiringMicros.r",
+        "locks.MMAPV1Journal.timeAcquiringMicros.w",
+        "locks.oplog.acquireCount.R",
+        "locks.oplog.acquireCount.w",
+        "locks.oplog.acquireWaitCount.R",
+        "locks.oplog.acquireWaitCount.w",
+        "locks.oplog.timeAcquiringMicros.R",
+        "locks.oplog.timeAcquiringMicros.w",
+
+        "backgroundFlushing.average_ms",
+        "backgroundFlushing.flushes",
+        "backgroundFlushing.last_ms",
+        "backgroundFlushing.total_ms",
```

Comment:
datadog has great support for monotonically increasing stats with their rate() function and the like